deployment/flatpak/usage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Flatpak Usage
+subsection: flatpak
+section: deployment
+---
+
+# {{page.title}}
+
+## Installing Flatpak Applications
+
+### Setting up the Flathub repository: 
+
+A team of Flatpak developers have started a project known as [Flathub](https://flathub.org). Flathub aims to provide a centralized repository for making Flatpak applications available to users. Flathub covers more than the GNOME application suite, and is regularly adding new applications.
```

Comment:
This is copied from external source, right? I'm not sure this is the right way- maybe just point to it and in one your own sentence explain like 'you can get the comminuty flatpacks from....`.

Reply:
This was taken from the FedoraMagazine article, yes.

Alright, I'll make the necessary changes.

Reply:
No need to copy whole blocks of text I'd say. See my other comment.